python/fledge/services/core/api/plugins/update.py

```diff
@@ -342,10 +339,11 @@ async def _get_sch_id_and_enabled_by_name(name: str) -> list:
     return result['rows']
 
 
-async def _put_schedule(protocol: str, host: str, port: int, sch_id: uuid, is_enabled: bool) -> None:
-    management_api_url = '{}://{}:{}/fledge/schedule/{}/enable'.format(protocol, host, port, sch_id)
+async def _put_schedule(host: str, port: int, sch_id: uuid, is_enabled: bool) -> None:
+    # Scheme is always http:// on core_management_port
+    management_api_url = 'http://{}:{}/fledge/schedule/{}/enable'.format(host, port, sch_id)
     headers = {'content-type': 'application/json'}
-    verify_ssl = False if protocol == 'HTTP' else True
+    verify_ssl = False
```

Comment:
Yes, this check has no impact; and actually was wrong

Reply:
Though, verify SSL matters only if HTTPS

Reply:
Yes, As we use self signed certificate, so if that comes into force via protocol then we want to set it to False. with HTTP its setting is useless.